code/extensions/FluentExtension.php

```diff
@@ -493,21 +505,20 @@ public static function set_enable_write_augmentation($enabled)
      *
      * @param string $condition Condition SQL string
      * @param array $includedTables
-     * @param string $locale Locale to localise to
-     * @return string Column identifier in "table"."column" format if it exists in this condition
+     * @return array Array with items [$table, $column], or null if not found
```

Comment:
I haven't standardised declaring null return types explicitly; I find it simpler to assume it's a possible response, and then just declare the non-null return types.

I might be doing it wrong though... but I could be convinced to adjust my practices. :)